src/oc_erchef/apps/chef_index/src/chef_solr.erl

```diff
@@ -44,22 +107,32 @@ make_query_from_params(ObjType, QueryString, Start, Rows) ->
     FilterQuery = make_fq_type(ObjType),
     %% 'sort' param is ignored and hardcoded because indexing
     %% scheme doesn't support sorting since there is only one field.
-    Sort = "X_CHEF_id_CHEF_X asc",
+    Sort = binary_to_list(id_field()) ++ " asc",
```

Comment:
Given the comment about the parameter being ignored - is it not ignored in cloudsearch? And is it really ignored in solr? 

I could see enforcing a sort as a means to get consistent ordering when paging, but am just curious if it's respected by one, both or neither provider(s).